website/pages/docs/upgrading/instructions/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+page_title: Upgrade Instructions
+sidebar_title: Upgrade Instructions
+description: >-
+  Specific versions of Consul may have additional information about the upgrade
+  process beyond the standard flow.
+---
+
+# Upgrade Instructions
+
+This document is intended to help customers who find themselves many versions behind to upgrade safely.
+Our recommended upgrade path is moving from version 0.8.5 to 1.2.4 to 1.6.9 to the current version. To get
+started, you will want to choose the version you are currently on below and then follow the instructions
+until you are on the latest version. The upgrade guides will mention notable changes and link to relevant
+changelogs – we recommend reviewing the changelog for versions between the one you are on and the
+one you are upgrading to at each step to familiarize yourself with changes.
+
+## Getting Started
+
+To get instructions for your upgrade, please choose the release series you are _currently using_:
+
+- [0.8.x](/docs/upgrading/instructions/upgrade-to-1-2-x)
+- [0.9.x](/docs/upgrading/instructions/upgrade-to-1-2-x)
+- [1.0.x](/docs/upgrading/instructions/upgrade-to-1-2-x)
+- [1.1.x](/docs/upgrading/instructions/upgrade-to-1-2-x)
+- [1.2.x](/docs/upgrading/instructions/upgrade-to-1-6-x)
+- [1.3.x](/docs/upgrading/instructions/upgrade-to-1-6-x)
+- [1.4.x](/docs/upgrading/instructions/upgrade-to-1-6-x)
+- [1.5.x](/docs/upgrading/instructions/upgrade-to-1-6-x)
+- [1.6.x](/docs/upgrading/instructions/upgrade-to-1-8-x)
+- [1.7.x](/docs/upgrading/instructions/upgrade-to-1-8-x)
+
+If you are using <= 0.7.x, please [contact support](https://support.hashicorp.com) for assistance.
```

Comment:
Since this page is meant for open source users and customers, this language may make open source users go to hashicorp support pages. I'd recommend removing this sentence